subsys/fb/cfb_fonts8x8.c

```diff
@@ -0,0 +1,1274 @@
+/*
```

Comment:
I do not understand why you refuse to use `scripts/gen_cfb_font_header.py`, especially with the improvements from @pabigot . It would improve the maintainability and make it easier to go away with the build-in fonts in general and generate the fonts at the build time.

Reply:
when I put my font there, I am not aware about @pabigot's commit, and pre his comments it is not obligated to use that. and generate fonts at build time is demoed in the cfb_custom_font sample. I put a standard font here just to want easy customer's usage. @jfischer-phytec-iot  so now I must use pabigot's script to generate my font header?

Reply:
Please take a look at #17541, it makes possible to generate htiled fonts as yours `cfb_fonts8x8.c`

Reply:
@jfischer-phytec-iot It doesn't right now; at your request I removed hpack support because there's no support for it in the CFB driver.  If #17541 gets merged I can add hpack support in another PR, coordinating with material in this PR.

Reply:
ok